service/routes.py

```diff
@@ -89,3 +91,76 @@ def create_shopcart():
 ######################################################################
 # I T E M   A P I S
 ######################################################################
+
+
+
+######################################################################
+# READ A SHOPCART
+######################################################################
+@app.route("/shopcarts/<int:shopcart_id>", methods=["GET"])
+def get_shopcarts(shopcart_id):
+    """
+    Retrieve a single Shopcart
+    This endpoint will return an Shopcart based on its id
+    """
+    app.logger.info("Request for Shopcart with id: %s", shopcart_id)
+
+    # See if the shopcart exists and abort if it doesn't
+    # try:
```

Comment:
I would vote for uncommenting line 109-112 😛

Reply:
If you're worrying about the test case, please refer to this commit: https://github.com/CSCI-GA-2820-SU23-001/shopcarts/pull/37/commits/f03863a54c835ecacc44cc49a7e544ba54e0890a

The two would be almost the same because they're mocking the same issue.

Reply:
however, give this is not an requirement, it is fine if you leave it as is 🙂

Reply:
There are some bugs when I use this try-catch structure when testing. I think maybe we can add if we have enough time after all the APIs are implemented.

Reply:
OK